src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -975,12 +1370,89 @@ export class SavedObjectsRepository {
     const savedObject = this._serializer.rawToSavedObject(raw);
     return omit(savedObject, 'namespace');
   }
+
+  private _rawInNamespaces(raw: SavedObjectsRawDoc, namespace?: string) {
```

Comment:
I'll change it to `rawDocExistsInNamespace`, I think that best describes how it works (since "`includesNamespace`" implies that it is a multi-namespace saved object, which it may not be).

Reply:
Updated in https://github.com/elastic/kibana/pull/54605/commits/e2f5d6573c45086c5dc1bbca045228590e35fa0b.